Ports/RISC-V/RV32/GCC/os_cpu_a.S

```diff
@@ -43,6 +43,7 @@
     .global  OSCtxSw
```

Comment:
There is this line at the top
"Hardware FP is not supported."

Later when we add floating point support, make sure to remove this line